client/src/pages/Home.js

```diff
@@ -19,6 +20,7 @@ export function Home() {
 			<div className="whats-new">
 				<WhatsNewCard />
 				<WhatsNewCard />
+				<Description />
```

Comment:
Description is not the right name for this component, maybe CreateEnergizer would be more descriptive